redash/handlers/authentication.py

```diff
@@ -196,10 +196,13 @@ def base_href():
 def date_format_config():
```

Comment:
Perhaps this could be renamed to `date_time_format_config`

Reply:
fixed in https://github.com/getredash/redash/pull/3754/commits/09de0b7ef49da369844078f0d9601fc47387d828